### PR TITLE
nunaliit-integration/5/link points

### PR DESCRIPTION
**Pull request by alexgao1**

### CODE REVIEW

File: nunaliit2-js/src/main/nunaliit-es6/src/n2es6/n2mapModule/N2LinkSource.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+					isFeatureVisible: isFeatureCurrentlyShowing,
+					i: point.data._ldata.timeLinkTags.placeTag // Not used but helpful for a human to read the place
+				}]);
+			}
+		});
+	}
+
+	_generateLinkStrengths() {
+		let previousData = [];
+		this._timeCoordinateData.forEach((currentData, _) => {
```
Comment (billingb) [inline nunaliit2-js/src/main/nunaliit-es6/src/n2es6/n2mapModule/N2LinkSource.js:76]: I think _ is unused and unneeded.

File: nunaliit2-js/src/main/nunaliit-es6/src/n2es6/n2mapModule/N2LinkSource.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+			for (let i = 0; i < previousData.length; i++) {
+				for (let j = 0; j < currentData.length; j++) {
+					const previousDataPoint = previousData[i];
```
Comment (alexgao1) [inline nunaliit2-js/src/main/nunaliit-es6/src/n2es6/n2mapModule/N2LinkSource.js:79]: Move line 79 and 82 out of this inner loop

File: nunaliit2-js/src/main/nunaliit-es6/src/n2es6/n2mapModule/N2LinkSource.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+	_collectTimeCoordinateData() {
+		this._pointFeatures.forEach(point => {
+			const featureStartTime = point.data._ldata.start;
+			const isFeatureCurrentlyShowing = point.get("isVisible");
+			const pointCoordinates = [...point.getGeometry().flatCoordinates];
+
+			if (this._timeCoordinateData.has(featureStartTime)) {
+				let pointData = this._timeCoordinateData.get(featureStartTime);
+				pointData.push({
```
Comment (alexgao1) [inline nunaliit2-js/src/main/nunaliit-es6/src/n2es6/n2mapModule/N2LinkSource.js:58]: Object can be made once instead of twice

**Review by billingb — APPROVED** [review]
Merge looks fine. Some small issues that could be improved, but code appears to be functionally correct.

**Review by billingb — APPROVED** [review]
Looks good

**Pull request merged**